TestAssets/TestProjects/LightupLib/project.json

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
> I think the "light up" library should bring in a new dependency that the "LightupClient" doesn't have.

This is what LightupClient is doing - it is bringing in LightLib that it does not have a reference to.